src/user-card/card.tsx

```diff
@@ -33,11 +37,23 @@ export interface UserCardWording {
 }
 export interface UserCardProps extends HTMLAttributes<HTMLDivElement> {
   user: UserCardUser
-  wording: UserCardWording
+  wording?: UserCardWording | null | undefined // TODO: "wording" isn't the right name, to be replaced with "translations" in 6.0
```

Comment:
What about using JSDoc `@deprecated` tag in such cases?
```suggestion
  /** @deprecated will be replaced with `translations` in 6.0 */
  wording?: UserCardWording | null | undefined
```
<img width="492" alt="image" src="https://user-images.githubusercontent.com/153412/235193395-83e52fcf-96d1-4b29-a7ae-4b23e76bc707.png">